packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -133,6 +146,32 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
     [onFetchAvatarPersonaData]
   );
 
+  /* @conditional-compile-remove-from(stable): FILE_SHARING */
+  const uploadedFilesSelector = useSelector(fileUploadsSelector);
+
+  /* @conditional-compile-remove-from(stable): FILE_SHARING */
+  const onRenderAttachedFiles = (): JSX.Element => {
+    const uploadedFiles = uploadedFilesSelector.files;
+    const truncateLength = 15;
+    return (
+      <FileCardGroup>
+        {uploadedFiles &&
+          uploadedFiles.map((file) => (
+            <FileCard
+              fileName={truncatedFileName(file.filename, truncateLength)}
+              progress={file.progress}
+              key={file.id}
+              fileExtension={extension(file.filename)}
+              actionIcon={<Icon iconName="Cancel" />}
```

Comment:
We should register this icon with a new name so Contoso can override it. Can be done in a future PR.

Reply:
@prabhjot-msft Let's create a task for this.